### PR TITLE
 finish list favorite movie

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
link redmi : https://edu-redmine.framgia.vn/issues/6840
 finish list favorite movie

### CODE REVIEW

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/java/com/example/moviedb_37/data/source/MovieDataSource.java [hunk 1]
```diff
@@ -10,13 +10,15 @@
 public interface MovieDataSource {
 
     interface Local extends MovieDataSource {
-        Single<List<Movie>> getFavoriteMovies();
+        List<Movie> getFavoriteMovies();
 
         boolean addFavariteMovie(Movie movie);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/data/source/MovieDataSource.java]: wrong name

File: app/src/main/java/com/example/moviedb_37/data/source/MovieDataSource.java [hunk 2]
```diff
@@ -10,13 +10,15 @@
...
 
         boolean deleteFavoriteMovie(Movie movie);
 
         boolean canAddFavarite(Movie movie);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/data/source/MovieDataSource.java]: tương tự

File: app/src/main/java/com/example/moviedb_37/data/source/MovieDataSource.java [hunk 3]
```diff
@@ -10,13 +10,15 @@
...
+
+        boolean canAddFavarite(int movieId);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/data/source/MovieDataSource.java]: tương tự

**Comment by meoconlontonbebong** [comment]
ready

**Pull request merged**